### ISSUE TITLE
[Story] As a user, I want to use template to send an email 

### ISSUE THREAD

**Issue by dieptran88** [linagora/tmail-flutter#1263]
# Summary

* [Related EPIC](#related-epic)
* [Definition](#definition)
* [Screenshots](#screenshots)
* [Misc](#misc)

## Related EPIC

* TODO: {link}

## Definition

- Given that I am a Tmail user
- I logged-in successfully to Tmail

**UC1. Use template to create an message**
- I go to mailbox "Template" and see the list of all templates 
- When I hover an template, the actions list will be displayed, there will be a new action: "Use template" 
- Or when I open the template detail, I can see the option "Use template" 
- When I select this option, the composer will be opened, with all the content from selected template
- I can continue edit the message and send as normally. 
- After sending the email using this template, the template is not changed 
- If I close email when composing, it will be saved to Draft and the template is not changed. 

[Back to Summary](#summary)

## Screenshots

![Group 173](https://github.com/linagora/tmail-flutter/assets/68209176/e9932007-9b45-4695-9d1d-f75bebca9214)

![Group 174](https://github.com/linagora/tmail-flutter/assets/68209176/6144ca6b-0aa6-415e-a6ae-5d825798d5e1)

[Back to Summary](#summary)

## Misc

None

[Back to Summary](#summary)

**Comment by dieptran88** [comment]
Note: We can improve with selecting template directly from the composer

**Comment by hoangdat** [comment]
> Note: We can improve with selecting template directly from the composer

good idea, how about put it in the toolbox ? But please keep the mobile version simple as possible

<img width="815" alt="Screenshot 2022-12-12 at 16 55 54" src="https://user-images.githubusercontent.com/6462404/207016041-9ac5e77f-26ab-4665-b219-154031614c64.png">

**Comment by chibenwa** [comment]
Template mailbox + edit as new does exactly this!
